news/2025/2025-06-15-project-loved-june-2025.md

```diff
@@ -0,0 +1,558 @@
+    <source src="https://assets.ppy.sh/project-loved/2025/2025-06-osu.mp4" type="video/mp4" preload="none">
+  </video>
+</div>
+
+---
+
+[![](/wiki/shared/news/2025-06-15-project-loved-june-2025/876611.jpg)](https://osu.ppy.sh/community/forums/topics/2092763)
+
+[Chaqueno Palavecino - La Ley y La Trampa](https://osu.ppy.sh/beatmapsets/876611#osu) by [MaestroSplinter](https://osu.ppy.sh/users/6707918)\
+117.75 BPM, 2:42 | 5.35★\
+*description written by [momoyo](https://osu.ppy.sh/users/12469536)*
+
+*Disclaimer: This description was written by Mismagius, a Brazilian. Opinions about Argentina may be biased.*
+
+Okay guys, I think we can all agree that Pelé was better than Maradona, right? Yeah, thought so.
+
+In all seriousness, here's [Chaqueño Palavecino - La Ley y La Trampa](https://osu.ppy.sh/beatmapsets/876611#osu), a classic Argentinian folk song by renowned mapper [MaestroSplinter](https://osu.ppy.sh/users/6707918). While this is a fairly standard map, it's still high quality. It's no wonder that it has over 500,000 playcount and over 700 favourites!
+
+MaestroSplinter's map is very good, I really love the patterns and jumps of the map. However, I don't like the fact that there are no streams since I bought a Rapid Trigger keypad, and I wanted to test it. I can't play the map with Double Time, but it's very fun with Hard Rock. Vote yes if you want an Argentinian song in the Loved section!
+
+As a tribute to this map, I shall proceed to write in Spanish by myself, without the use of any translation tools or corrections, because I know my Spanish is perfect.
+
+El mapa de MaestroSplinter es muy bueno, me encantan mucho las patronas y los saltos del mapa. Pero no me gusta el facto que no tiene arroyos ya que compré un keypad Gatillo Rapido y me gustaría testarlo. No puedo jugar al mapa con Doble Tiempo pero es muy divertido con Roca Dura. Vota sí si quieres una canción argentina en la sección Amada!
+
+[![](/wiki/shared/news/2025-06-15-project-loved-june-2025/1261907.jpg)](https://osu.ppy.sh/community/forums/topics/2092762)
+
+[inabakumori - Lost Umbrella](https://osu.ppy.sh/beatmapsets/1261907#osu) by [Chills](https://osu.ppy.sh/users/13918392)\
+274 BPM, 3:17 | 2.46★, 3.76★, 4.52★, 5.41★, 6.36★\
+*description written by [momoyo](https://osu.ppy.sh/users/12469536)*
+
+Lost Umbrella? I got you!!
+
+Coming in with the Popular Playcounts this month, we got a good classic song with a very known graveyard set that deserves more recognition... The comments also do not lie this map is RLLY fun to play, which is something that made this mapset popular back in the day it was submitted.
+
+With a solid 500K plays and over 450 favorites, and also a full set from 2 to 6 stars difficulties, I'm proudly bringing up this set for the Loved section for those who either played this before and/or wish to see it with a leaderboard!!
+
+[![](/wiki/shared/news/2025-06-15-project-loved-june-2025/184686.jpg)](https://osu.ppy.sh/community/forums/topics/2092761)
```

Comment:
no bg?

Reply:
web has no bg hence it used default

actually lemme try regenerating covers from web i wonder if that'd work

Reply:
it did!!